ghcloneall.py

```diff
@@ -929,8 +929,11 @@ def _main():
                     CONFIG_FILE))
         return
 
+    if args.include_private and not args.github_token:
+        print('Warning: Listing private repositories requires a GitHub token')
```

Comment:
The warning probably ought to go to sys.stderr.

Another thing I need to double-check: what happens if you run

    ghcloneall --init --user foo

with an _old_ version of ghcloneall?  I'm 75% sure it'll create a .ghcloneallrc with `include_private = True` in it, which will make _new_ versions of ghcloneall emit this warning.

I'd prefer if users wouldn't have to manually update config files to avoid new warnings, but that wish may be incompatible with a wish for there to be a warning in the first place.  Hmm, I may need to think on this more.

Reply:
With the HEAD of my branch I get:

```
$ ghcloneall --init --user foo
Wrote .ghcloneallrc
$ cat .ghcloneallrc
[ghcloneall]
github_user = foo
```

With 1.10.1 I get the same

```
$ ghcloneall --init --user foo
Wrote .ghcloneallrc
$ cat .ghcloneallrc
[ghcloneall]
github_user = foo
```

1.9.2 produces the same.

So, I think we're OK?

Reply:
> The warning probably ought to go to sys.stderr.

Done